releasenotes/notes/jax-sparse-solvers-8d469991f71466b0.yaml

```
@@ -0,0 +1,15 @@
+---
+feature:
+  - |
+    Adds JAX sparse evaluation mode to models and
+    :class:`~qiskit_dynamics.solvers.solver_classes.Solver`.
```

For this it would be good to show the full import path a user would import from rather than the file path:
```suggestion
    :class:`qiskit_dynamics.Solver`.
```